.mci.yml

```diff
@@ -107,7 +121,32 @@ buildvariants:
       display_name: "Ubuntu 14.04 Debug"
       expansions:
           build_type: "Debug"
+          source: *linux_source
+          tar_options: *linux_tar_options
+          cmake_path: *linux_cmake_path
       run_on:
           - ubuntu1404-test
       tasks:
           - name: compile_and_test
+    - name: osx-108
```

Comment:
No, I think one or the other is sufficient. I don't have a strong opinion on which.